### PR TITLE
New vWAN resource properties

### PR DESCRIPTION
**Pull request by henry416**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

+ Added DPDTimeout for VpnSiteLinkConnection (bring vWAN to parity with VNG)
+ Added IP Configurations for VPNGateway

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by MyronFanQiu** [comment]
@henry416 Usually we don't send PR targeting at master branch directly. @number213 Please confirm it.

**Comment by pilor** [comment]
> @henry416 Usually we don't send PR targeting at master branch directly. @number213 Please confirm it.

Agreed.  Holding off on reviewing until there is confirmation that this is the right target branch

**Comment by henry416** [comment]
@number213 Please let me know which branch to target

**Comment by henry416** [comment]
@number213 please review

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json [hunk 1]
```diff
@@ -4301,6 +4301,13 @@
           "type": "integer",
           "format": "int32",
           "description": "The scale unit for this vpn gateway."
+        },
+        "ipConfigurations": {
```
Comment (majastrz) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:4305]: Is this a read-only or read/write property?

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json [hunk 2]
```diff
@@ -4486,6 +4510,11 @@
           "format": "int32",
           "description": "Routing weight for vpn connection."
         },
+        "dpdTimeoutSeconds": {
```
Comment (majastrz) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:4513]: This should be added in a new API version.

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json (hunk 1 above)
Comment (majastrz) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:4305]: Can you mark it as read-only in the swagger?

---
In reply to: [438295999](https://github.com/Azure/azure-rest-api-specs/pull/9722#discussion_r438295999) [](ancestors = 438295999)

**Comment by number213** [comment]
Please retarget to `network-june-release `

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json (hunk 1 above)
Comment (henry416) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:4305]: @majastrz Done

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json (hunk 2 above)
Comment (henry416) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:4513]: @majastrz Done

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json (hunk 1 above)
Comment (majastrz) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:4305]: I don't see it.

---
In reply to: [439132364](https://github.com/Azure/azure-rest-api-specs/pull/9722#discussion_r439132364) [](ancestors = 439132364)

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json (hunk 2 above)
Comment (majastrz) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json:4513]: Thanks!

---
In reply to: [439132408](https://github.com/Azure/azure-rest-api-specs/pull/9722#discussion_r439132408) [](ancestors = 439132408)

**Review by majastrz — APPROVED** [review]
Signed off from ARM side since API shape looks good to me. However, please make sure the new property is actually marked as read only in the Swagger.

File: specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json (hunk 1 above)
Comment (henry416) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json]: @majastrz Moved to 2020-06 version

Comment (majastrz) [reply specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json]: I definitely see the move to the June API version. I mean I don't see the read-only part, which you said was "Done".

**Comment by henry416** [comment]
@MyronFanQiu Please review, no breaking changes are introduced as these are optional fields

**Comment by henry416** [comment]
@MyronFanQiu @number213 @v-shhatt Please review

**Review by mmyyrroonn — APPROVED** [review]

**Pull request merged**